netket/operator/_kinetic.py

```diff
@@ -0,0 +1,67 @@
+from typing import Optional
+
+from netket.utils.types import DType, PyTree
+
+from netket.hilbert import AbstractHilbert
+from netket.operator import ContinousOperator
+
+import jax
+import jax.numpy as jnp
+
+
+class KineticEnergy(ContinousOperator):
+    def __init__(
+        self, hilbert: AbstractHilbert, mass: PyTree, dtype: Optional[DType] = float
```

Comment:
why mass is a pytree and not a simple array or just a scalar ? I would make it scalar (meaning all particles have the same mass) by default or if wanted an array, for different masses